apps/send-func/src/adapters/send/definitions.ts

```diff
@@ -173,3 +173,16 @@ export const sendHeadersSchema = lollipopHeadersSchema.merge(
 );
 
 export type SendHeaders = z.TypeOf<typeof sendHeadersSchema>;
+
+export const sendAARClientResponseSchema = z.object({
+  jsonBody: z.union([
+    checkQrMandateResponseSchema,
+    problemJsonSchema,
+    thirdPartyMessageSchema,
+  ]),
+  status: z.number().int(),
```

Comment:
Can we use a literal here? We can pass any int here and it would be good for the compiler

Reply:
Yes, we should only allow status codes the openapi expose